src/helpers/getTheme.ts

```diff
@@ -1,8 +1,18 @@
 import { get } from 'lodash';
+import { pxToRem } from './pxToRem';
 import { FluidTheme, ThemeProps } from '../index';
 
+enum TypeValue {
+  rem = 'rem'
+}
+
 export const getTheme =
-  (themeProp: Flatten<FluidTheme>) =>
+  (themeProp: Flatten<FluidTheme>, options?: { format: keyof typeof TypeValue }) =>
   ({ theme }: ThemeProps): string | number | null => {
-    return get(theme, themeProp) || null;
+    const parameter = get(theme, themeProp) || null;
+    if(options?.format === TypeValue.rem){
```

Comment:
@mdlucas o que acha de em vez de fazer isso manualmente em todas as vezes que for utilizar o helper, ele detectar o ambiente e automaticamente já usar rem, e esse argumento deixa a opção de forçar a não usar rem? 🤔

Reply:
feito